### ISSUE TITLE
`createStep` called from JS thrown error if `appendToBody` is false

### ISSUE THREAD

**Issue by balazsnemeth** [benmarch/angular-ui-tour#153]
**I am using the _exact_ versions of the following:**

* Browser: Chome Version: 61.0
* AngularJS: 1.6.1
* Bootstrap: "3.3",
* Angular Bootstrap: 2.5
* Angular UI Tour: 0.9.0

**I have installed this library via:** Bower

**I have observed the following behavior:**

When I register a step using `createStep` by specifying a `selector`, the following line thrown an error in `createPopup(step, tour)`:

```js
parent = step.config('appendToBody') ? _angular2.default.element($document[0].body) : step.element.parent();
```

The error:
<img width="495" alt="screenshot 2017-09-25 15 59 42" src="https://user-images.githubusercontent.com/2070044/30828153-30c924a4-a1e9-11e7-9f4d-fdc412dec129.png">

**Here is my tour config, and all related step configs:**

```js
var step =  {
    selector: 'a[ui-sref="main.test"]', // This type of step will be loaded dynamically!
    order: 40,
    title: 'Test Title',
    placement: 'bottom',
    content: 'Lorem ipsum dolor sit amet, pro cu brute accommodare, vis ad minim eripuit cotidieque',
  }

...

tour.createStep(step);
```

**Comment by benmarch** [comment]
Thank you @balazsnemeth, I will fix this for the next release.

**Comment by benmarch** [comment]
Released a potential fix for this in 0.9.1. Please let me know if it works.
